yucca/training/lightning_modules/YuccaLightningModule.py

```diff
@@ -177,7 +187,13 @@ def configure_optimizers(self):
 
         loss_kwargs = filter_kwargs(self.loss_fn, loss_kwargs)
 
-        self.loss_fn = self.loss_fn(**loss_kwargs)
+        self.loss_fn_tr = self.loss_fn(**loss_kwargs)
```

Comment:
I really do not like the `tr` ending since it is too close to test. I would write it out so it becomes `loss_fn_train`. Three letters, but makes a huge difference to me.